lib/keyboard_utils.coffee

```diff
@@ -60,6 +62,7 @@ KeyboardUtils =
       if (event.keyCode >= @keyCodes.f1 && event.keyCode <= @keyCodes.f12)
         return "f" + (1 + event.keyCode - keyCodes.f1)
       return ""
+    return "backspace" if event.keyIdentifier == "U+0008"
```

Comment:
No, because `keyNames` can only be searched `if (event.keyIdentifier.slice(0, 2) != "U+")`